src/main/java/com/redhat/devtools/lsp4ij/LanguageServerWrapper.java

```diff
@@ -106,6 +128,8 @@ record LSPFileConnectionInfo(@Nullable Document document, @Nullable String docum
 
     private final ExecutorService listener;
 
+    private final SimpleModificationTracker modificationTracker = new SimpleModificationTracker();
```

Comment:
Wrapper-level modification tracker for server status changes. This is added/used once servers have been started and is complementary to the project-specific, server-agnostic modification tracker in `LanguageServerAccessor`.